ledger/participant-integration-api/src/main/scala/platform/store/backend/localstore/ResourceVersionOpsBackend.scala

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.store.backend.localstore
+
+import java.sql.Connection
+
+trait ResourceVersionOpsBackend {
```

Comment:
That class is in a different category to all the other back-ends. I suggest to call it just `ResourceVersionOps`

Reply:
Ok